Signal/src/ViewControllers/UpdateGroupViewController.m

```diff
@@ -0,0 +1,504 @@
+                            @"CONTACT_CELL_IS_BLOCKED", @"An indicator that a contact has been blocked.");
+                    } else {
+                        cell.selectionStyle = UITableViewCellSeparatorStyleNone;
+                    }
+                } else {
+                    // In the "members" section, we label "new" members as such when editing an existing group.
+                    //
+                    // The only way a "new" member could be blocked is if we blocked them on a linked device
+                    // while in this dialog.  We don't need to worry about that edge case.
+                    cell.accessoryMessage = NSLocalizedString(
+                        @"EDIT_GROUP_NEW_MEMBER_LABEL", @"An indicator that a user is a new member of the group.");
+                }
+
+                if (signalAccount) {
+                    [cell configureWithSignalAccount:signalAccount contactsManager:helper.contactsManager];
+                } else {
+                    [cell configureWithRecipientId:recipientId contactsManager:helper.contactsManager];
+                }
+
+                return cell;
+            }
+                        customRowHeight:[ContactTableViewCell rowHeight]
+                        actionBlock:^{
+                            SignalAccount *signalAccount = [helper signalAccountForRecipientId:recipientId];
+                            BOOL isPreviousMember = [weakSelf.previousMemberRecipientIds containsObject:recipientId];
+                            BOOL isBlocked = [helper isRecipientIdBlocked:recipientId];
+                            if (isPreviousMember) {
+                                if (isBlocked) {
+                                    if (signalAccount) {
+                                        [weakSelf showUnblockAlertForSignalAccount:signalAccount];
+                                    } else {
+                                        [weakSelf showUnblockAlertForRecipientId:recipientId];
+                                    }
+                                }
+                            } else {
+                                if (signalAccount) {
+                                    [weakSelf.groupViewHelper
+                                        showRemoveFromGroupAlertForSignalAccount:signalAccount
+                                                              fromViewController:weakSelf
+                                                                 contactsManager:helper.contactsManager
+                                                                    successBlock:^{
+                                                                        [weakSelf removeSignalAccount:signalAccount];
+                                                                    }];
+                                } else {
+                                    [weakSelf.groupViewHelper
+                                        showRemoveFromGroupAlertForRecipientId:recipientId
```

Comment:
Good point. I've removed remove confirmation alerts from the "new group" and "update group" views.